plugins/kubernetes/app/models/kubernetes/role_validator.rb

```diff
@@ -103,6 +104,13 @@ def self.keep_name?(e)
 
     private
 
+    def validate_load_balancer
+      allowed = ENV["KUBERNETES_ALLOWED_LOAD_BALANCER_NAMESPACES"].to_s.split(",")
+      return if allowed.empty?
+      return if @elements.none? { |e| e[:kind] == "LoadBalancer" && !allowed.include?(e.dig(:metadata, :namespace)) }
+      @errors << "LoadBalancer is not allowed in this namespace"
```

Comment:
It would be great if you print the namespace as well in which it is not allowed, so it will be clear.

Reply:
done